src/main/java/com/ait/lienzo/client/core/animation/LayerRedrawManager.java

```diff
@@ -65,18 +65,17 @@ public final Layer schedule(final Layer layer)
             if (false == m_layers.contains(layer))
             {
                 m_layers.add(layer.doBatchScheduled());
-
-                kick();
+                kick(layer.getElement());
             }
         }
         return layer;
     }
 
-    private void kick()
+    private void kick(Element layerElement)
```

Comment:
For consistency with other method signatures in this class we should declare this one as:
```java
private void kick(final Element layerElement)
{
// ...
}
```